src/app/base/components/MainContentSection/MainContentSection.tsx

```diff
@@ -23,11 +24,14 @@ const MainContentSection = ({
   const { SIDEBAR, TOTAL } = COL_SIZES;
   return (
     <div {...props} id="main-content-section">
-      <div>
+      <ContentSection>
```

Comment:
Once we add this, we'll also need to remove duplicated `ContentSection` wrapper from settings pages.

Reply:
Which leaves a question - what feels more natural - handling this globally in PageContent component - or individually per page?

On one hand, handling it per page definitely gives us more flexibility but that can also lead to inconsistencies in the future.

@ndv99 How do you think?

Reply:
I think I'm leaning more towards handling it globally. It may be less flexible, but given that pages in maas-ui follow a very similar layout to each other, I think it makes sense here and reduces repetition.

Reply:
Same here!